src/olympia/reviewers/tests/test_utils.py

```diff
@@ -3708,6 +3767,44 @@ def test_reject_multiple_versions_resets_original_status_too(self):
             File.STATUS_DISABLED_REASONS.NONE
         )
 
+    def test_request_legal_review_no_job(self, *, data=None):
+        self.setup_data(
+            amo.STATUS_APPROVED,
+            file_status=amo.STATUS_APPROVED,
+        )
+        if data:
+            data = {**self.get_data(), **data}
+            self.helper.set_data(data)
+        report_request = responses.add(
+            responses.POST,
+            f'{settings.CINDER_SERVER_URL}create_report',
+            json={'job_id': '5678'},
+            status=201,
+        )
+
+        self.helper.handler.request_legal_review()
+
+        assert len(mail.outbox) == 0
+        assert report_request.call_count == 1
+        assert CinderJob.objects.last().job_id == '5678'
+        request_body = json.loads(responses.calls[0].request.body)
+        assert request_body['reasoning'] == 'foo'
+
+    def test_request_legal_review_resolve_job(self):
```

Comment:
This test needs a doc string.. Because "resolvable_in_reviewer_tools is True, and because we call `test_request_legal_review_no_job (passing a job, odd) we then expect the job.forwarded_to_job to equal to the last created job, with no other criteria. It doesn't really add up to me. Like, I could say, sure it makes sense, but it's very fuzzy why this makes sense.

Reply:
It's... me ~being lazy~ exercising DRY with tests - because the functionality for escalating _without_ a job is a subset of the expected functionality for escalating with a job.  I can refactor to make it clearer...

Reply:
I'm not asking to change the test implementation, just explain why the test, as written, should pass. I'm all in favor of dry tests otherwise we would have millions more lines of code.

Reply:
I changed the test implementation because a clearer code is better than unclear code with an explanation, imo.

Reply:
That was ~~ironic~~ pythonic.